@here/harp-omv-datasource/test/OmvDataSourceTest.ts

```diff
@@ -143,4 +148,59 @@ describe("DataProviders", function() {
             assert.equal((omvDataSource.decoder as any).m_storageLevelOffset, 2);
         });
     });
+
+    it("DataProvider clears the cache", function() {
+        const geojson: FeatureCollection = {
+            type: "FeatureCollection",
+            features: []
+        };
+
+        const markTilesDirty = sinon.fake();
```

Comment:
why do you need this? It doesn't seem to be used.

Reply:
it is needed to get the decoder registration working, without it the test fails

```
(node:37761) UnhandledPromiseRejectionWarning: TypeError: this.mapView.markTilesDirty is not a function
    at OmvDataSource.configureDecoder (harp.gl/@here/harp-omv-datasource/lib/OmvDataSource.ts:419:22)
```

Reply:
Ok, in that case you could make it inline, but minor..